rust/parquet/src/util/test_common/file_util.rs

```diff
@@ -19,11 +19,17 @@ use std::{env, fs, io::Write, path::PathBuf, str::FromStr};
 
 /// Returns path to the test parquet file in 'data' directory
 pub fn get_test_path(file_name: &str) -> PathBuf {
-    let result = env::var("PARQUET_TEST_DATA");
-    if result.is_err() {
-        panic!("Please point PARQUET_TEST_DATA environment variable to the test data directory");
-    }
-    let mut pathbuf = PathBuf::from_str(result.unwrap().as_str()).unwrap();
+    let mut pathbuf = match env::var("PARQUET_TEST_DATA") {
+        Ok(path) => PathBuf::from_str(path.as_str()).unwrap(),
+        Err(_) => {
+            let mut pathbuf = env::current_dir().unwrap();
```

Comment:
I think you need to allow for running from the root arrow directory (`arrow`) or the sub-crates (i.e. `arrow\parquet`) due to the use of work spaces.  So the number of pops would need to be adjusted, right?

Reply:
It seems that regardless of where I run it from, the current directory is always `arrow/rust/parquet`.

Reply:
@paddyhoran You've used `\` as delimiter, do you use Windows? If so, I'd appreciate if you could test the code locally :)

Reply:
Yep, I’m on windows. I’ll test locally tonight and report back.